css/amp.css

```diff
@@ -346,16 +346,15 @@ i-amp-scroll-container.amp-active {
 .i-amphtml-loader-moving-line {
   display: block;
   position: absolute;
-  left: -100%;
```

Comment:
I found `0% {transform: translateX(-100%);}` does the work. Is there case that 0% will not cover?